src/themes/masschallenge/globals/site.variables

```diff
@@ -16,6 +16,7 @@
 @textColor: #676767;
 @greyBackground: #efebeb;
 @whiteColor: #fff;
```

Comment:
**Not blocking** Noticing inconsistency in variable naming: All of these are colors but one is an `object + color identifier` pairing, one is a `color name + object pairing`, and the last is a `color name + color identifier pairing`. I'm not sure if this is worth making a fuss about as I don't have a strong opinion, but it does strike me as being a potential problem in the future if our list of color-based variables grows.